my/time/tz/common.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Literal, Callable
+
+from ...core.common import tzdatetime
+
+
+'''
+Depending on the specific data provider and your level of paranoia you might expect different behaviour.. E.g.:
+- if your objects already have tz info, you might not need to call localize() at all
+- it's safer when either all of your objects are tz aware or all are tz unware, not a mixture
+- you might trust your original tiemzone, or it might just be UTC, and you want to use something more reasonable
+'''
+Policy = Literal[
```

Comment:
Not sure about `Literal`, haven't really used it much before.. But in my experience having real enums for such things just makes it a bit annoying (e.g. you have to import it first, etc). I guess mypy gives enough safety while keeping it flexible.
Either way should be easy to switch in the future, so maybe good to give it a go anyway.